src/mlvis-common/__tests__/computation.spec.js

```diff
@@ -126,31 +144,38 @@ test('utils: computeFeatureMeta', () => {
   expect(computeFeatureMeta('catFeature', arr1)).toEqual({
     name: 'catFeature',
     type: FEATURE_TYPE.CATEGORICAL,
+    dataType: DATA_TYPE.INTEGER,
     domain: [1, 2, 3, 4, 5, 6, 7],
   });
   expect(computeFeatureMeta('catFeature', arr2)).toEqual({
     name: 'catFeature',
     type: FEATURE_TYPE.CATEGORICAL,
+    dataType: DATA_TYPE.STRING,
     domain: ['a0', 'a1'],
   });
   expect(computeFeatureMeta('nullFeature', arr3)).toEqual({
     name: 'nullFeature',
     type: null,
+    dataType: null,
```

Comment:
`DATA_TYPE.NULL` for consistency

Reply:
Removed the null value from the constant, for now.